packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -175,14 +291,80 @@ abstract class FlameChartState<T extends FlameChart, V> extends State<T>
       rows.add(FlameChartRow(i));
     }
   }
+
+  void _handleMouseEnter(PointerEnterEvent event) {
+    focusNode.requestFocus();
+  }
+
+  void _handleMouseExit(PointerExitEvent event) {
+    focusNode.unfocus();
+  }
+
+  void _handleMouseHover(PointerHoverEvent event) {
+    mouseHoverX = event.position.dx - flameChartContainerOffset;
+  }
+
+  void _handleKeyEvent(RawKeyEvent event) {
+    if (event.isShiftPressed) {
+      _shiftKeyPressed = true;
+    } else {
+      _shiftKeyPressed = false;
+    }
+
+    // Only handle down events so logic is not duplicated on key up.
+    if (event is RawKeyDownEvent) {
+      // Handle zooming / navigation from W-A-S-D keys.
+      final keyLabel = event.data.keyLabel;
```

Comment:
nit: why are we using wsad instead of the arrow keys? is that for compatibility with the observatory?

Reply:
WASD will be familiar for users with experience using other trace viewers (e.g. chrome://tracing, Observatory, etc.).  Wdyt about allowing both WASD and the arrow keys to zoom / pan?